Firebase/Auth/Source/SystemService/FIRAuthNotificationManager.m

```diff
@@ -107,12 +119,20 @@ - (void)checkNotificationForwardingWithCallback:(FIRAuthNotificationForwardingCa
         kNotificationProberKey : @"This fake notification should be forwarded to Firebase Auth."
       }
     };
+#if !TARGET_OS_WATCH
```

Comment:
Can you move this down below and make it a `#if TARGET_OS_WATCH`? That should make a cleaner diff as well.